spec/activerecord-multi-tenant/model_extensions_spec.rb

```diff
@@ -496,7 +496,12 @@
         SELECT  "projects".* FROM "projects" WHERE "projects"."id" = $1 AND "projects"."account_id" = #{account.id} LIMIT $2
       sql
 
-      expect(Project).to receive(:find_by_sql).with(eq(option1).or(eq(option2)).or(eq(option3)), any_args).and_call_original
+      # Couldn't make the following line pass for some reason, so came up with an uglier alternative
+      # expect(Project).to receive(:find_by_sql).with(eq(option1).or(eq(option2)).or(eq(option3)), *any_args).and_call_original
```

Comment:
```suggestion
      # expect(Project).to receive(:find_by_sql).with(eq(option1).or(eq(option2)).or(eq(option3)), any_args).and_call_original
```

Reply:
Probably kwargs is acting up & you needed something like `with(eq-expr, anything, preparable: anything)`